backend/clubs/migrations/0068_clubvisit_ip.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.1.4 on 2021-01-02 23:42
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("clubs", "0067_profile_show_profile"),
```

Comment:
You'll need to delete and regenerate this migration after Lucy merged in her PR, since you've both made database edits.

Reply:
Also, delete the `backend/.swp` file in this PR.